bibliopixel/drivers/WS2812SPI.py

```diff
@@ -0,0 +1,50 @@
+from . spi_driver_base import DriverSPIBase, ChannelOrder
+from .. import gamma as _gamma
+
+
+class DriverWS2812SPI(DriverSPIBase):
+    """SPI driver for WS2812(b) based LED strips on devices like the Raspberry Pi, OrangePi, BeagleBone,.."""
```

Comment:
`flake8 --select F401,E501,E402 bibliopixel/drivers/WS2812SPI.py`
has 0 as return value and no output